snippets/c.json

```diff
@@ -561,9 +561,9 @@
       "/*!",
       " * @brief ${4:This function is undocumented.}",
       " *",
-      " * ${5:Nothing to say here...}",
+      " * ${5:Nothing to add here...}",
       " *",
-      " * @return ${6:Nothing.}$0",
+      " * @return ${6:Umm...}$0",
```

Comment:
Perhaps have this be `${6: return type}`?

Reply:
Umm, not sure that `${6: return type}` means in your context... Like a placeholder?

If so, there's 2 reasons why I suggest doing this way.

1. [It's not about the type, it's about the values, e.g. success and error values.](https://www.doxygen.nl/manual/commands.html#cmdreturn)
2. And the main reason. Read about the `Details...` thing below.

Reply:
Ok so revisiting this now, I can now see where you were approaching this. I'm content with this change